### PR TITLE
Python packaging

### PR DESCRIPTION
**Pull request by phodina**
Hi,

here's also PR to add Python packaging to allow distros to install the tool easily.

### CODE REVIEW

**Review by steadfasterX — CHANGES_REQUESTED** [review]
other then with the kdztools I would not like to create a src dir here tbh. this will break any manual user guide and has no benefit other then the packaging itself. 
for kdztools I am fine as long as the requested changes are made but lglaf is different as it breaks the logic how it is used out there.

Merge of the toml and updated rules files would be not a problem though when you remove the dir movement

hope for your understanding and thanks for contributing

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+name = "lglaf"
+version = "0.1"
+authors = [
+{ name="Peter Wu" , email="peter@lekensteyn.nl"},
+{ name="steadfasterX" , email="steadfasterX@gmail.com"},]
+description="LG Download Mode utility"
+readme="README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+	"License :: OSI Approved :: MIT License",
```
Comment (phodina) [inline pyproject.toml:11]: Will fix the indent.

**Comment by phodina** [comment]
> other then with the kdztools I would not like to create a src dir here tbh. this will break any manual user guide and has no benefit other then the packaging itself. for kdztools I am fine as long as the requested changes are made but lglaf is different as it breaks the logic how it is used out there.
> 
> Merge of the toml and updated rules files would be not a problem though when you remove the dir movement
> 
> hope for your understanding and thanks for contributing

Well but IMHO the manual install is not the best way to use programs. Having it packaged and just adding the tool through the package manager has lot of benefits. Also then you update the package based on releases.

Do you want me to also update the guide for the users?

**Comment by steadfasterX** [comment]
> > other then with the kdztools I would not like to create a src dir here tbh. this will break any manual user guide and has no benefit other then the packaging itself. for kdztools I am fine as long as the requested changes are made but lglaf is different as it breaks the logic how it is used out there.
> > Merge of the toml and updated rules files would be not a problem though when you remove the dir movement
> > hope for your understanding and thanks for contributing
> 
> Well but IMHO the manual install is not the best way to use programs. Having it packaged and just adding the tool through the package manager has lot of benefits. Also then you update the package based on releases.
> 
> Do you want me to also update the guide for the users?

that's correct. ideally it would be both packaged but then I will loose e.g. the flexibility that I need to switch between versions etc. also it is included in mAid as a terminal only variant etc etc.
all that can be solved as well though but nothing I want be able to do - at least not atm. it will need also a lot of testing in several places and I cannot provide that atm.

**Comment by steadfasterX** [comment]
will close that PR now as it does not fit my requirements. Feel free to open a new one (pointing to develop branch always) which matches these. thanks again

**Pull request closed without merging**